python/paddle_fl/mpc/examples/lenet_with_mnist/train_lenet.py

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+MNIST CNN Demo (LeNet5)
+"""
+
+import sys
+import os
+import numpy as np
+import time
+import logging
+import math
+
+import paddle
+import paddle.fluid as fluid
+import paddle.fluid.profiler as profiler
+import paddle_fl.mpc as pfl_mpc
+import paddle_fl.mpc.data_utils.aby3 as aby3
+
+logging.basicConfig(format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger("fluid")
+logger.setLevel(logging.INFO)
+
+role, server, port = sys.argv[1], sys.argv[2], sys.argv[3]
+# modify host(localhost).
+pfl_mpc.init("aby3", int(role), "localhost", server, int(port))
+role = int(role)
+
+# data preprocessing
+BATCH_SIZE = 128
+epoch_num = 5
+
+x = pfl_mpc.data(name='x', shape=[BATCH_SIZE, 1, 28, 28], dtype='int64')
+y = pfl_mpc.data(name='y', shape=[BATCH_SIZE, 10], dtype='int64')
+
+# lenet-3 network
+#conv = pfl_mpc.layers.conv2d(input=x, num_filters=16, filter_size=5, act='relu')
+#pool = pfl_mpc.layers.pool2d(input=conv, pool_size=2, pool_stride=2)
+#fc_1 = pfl_mpc.layers.fc(input=pool, size=100, act='relu')
+
+# lenet-5 network
```

Comment:
U can define two network class here, one is Lenet3 and the other is Lenet5. Then choose one of them to minimize.

Reply:
fixed